settings.py

```diff
@@ -313,9 +312,7 @@
 if LOCAL_INSTALLED_APPS:
     INSTALLED_APPS.extend(LOCAL_INSTALLED_APPS)
 
-# SUPPORTED_REPO.extend(["bitbucket", "github", "gitlab"])
-SUPPORTED_REPO.extend(["bitbucket", "github", "codeberg"])
-
+SUPPORTED_REPO = ["bitbucket", "github", "codeberg"]
```

Comment:
I believe we need GitLab too. 

```suggestion
SUPPORTED_REPO = ["bitbucket", "github", "gitlab", "codeberg"]
```

Reply:
Interesting, why was gitlab not enabled before?

Reply:
Hmm, good point. I'll have to local test it.